modules/monitoring-configuring-alertmanager-to-send-notifications-for-default-platform-alerts.adoc

```diff
@@ -3,70 +3,76 @@
 // * observability/monitoring/managing-alerts.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="applying-custom-alertmanager-configuration_{context}"]
-= Applying a custom Alertmanager configuration
+[id="configuring-alertmanager-to-send-notifications-for-default-platform-alerts_{context}"]
+= Configuring the Alertmanager to send notifications for default platform alerts
 
-You can overwrite the default Alertmanager configuration by editing the `alertmanager-main` secret in the `openshift-monitoring` namespace for the platform instance of Alertmanager.
+You can customize the default platform instance of Alertmanager to send notifications by overwriting the default Alertmanager configuration. You can achieve this by editing the `alertmanager-main` secret in the `openshift-monitoring` namespace.
```

Comment:
Should the first sentence be more focused on the objective? E.g. the goal is to configure where alert notifications should be dispatched.
"You can customize where Alertmanager sends notifications about default platform alerts."
Also it's almost a SHOULD action for the users because the out-of-the-box config generates no notification. Maybe we should highlight this somewhere as a preamble?

Reply:
Hello @simonpasquier thank you for pointing this out! We have new ["getting started" docs](https://79966--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/common-monitoring-configuration-scenarios), I think that is the best place to highlight it (we already mention it for UWM, but not for default platform monitoring)

I will also add a note in this new section about its importance, as you just highlighted it :+1:

Reply:
:+1: